apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -57,23 +55,24 @@ const AmountField = ({
   const onChangeSendMax = useCallback(
     (useAllAmount: boolean) => {
       onChangeTransaction(
-        bridge.updateTransaction(transaction, { useAllAmount, amount: BigNumber(0) }),
+        bridge.updateTransaction(transaction, { useAllAmount, amount: new BigNumber(0) }),
       );
     },
     [bridge, transaction, onChangeTransaction],
   );
 
   if (!status) return null;
+
   const { useAllAmount } = transaction;
   const { amount, errors, warnings } = status;
-  let { amount: amountError, gasPrice: messageGas } = errors;
+  let { amount: amountError, dustLimit: messageDust, gasPrice: messageGas } = errors;
   let { amount: amountWarning } = warnings;
 
   // we ignore zero case for displaying field error because field is empty.
 
   if (amount.eq(0) && (bridgePending || !useAllAmount)) {
-    amountError = null;
-    amountWarning = null;
+    amountError = undefined;
```

Comment:
Don't think it would change anything here, probably just for typing, **but** (I know it was there before and probably out of scope of this PR) we should probably avoid changing the content of errors returned by the bridge in general and always consider them as constants.

To me it would be worth changing this piece of logic for a memoized constant with conditions inside taking into account this moment where `amount.eq(0)` and feed this/those variable(s) to the `RequestAmount` component.  

TBD if we want this in this PR 🤷‍♂️

Reply:
Agreed but I think it's a better idea to do this in a different PR